### PR TITLE
Update/js ref functions af

### PR DESCRIPTION
**Pull request by undead404**
Оригінальний вміст: [Arrow function expressions@MDN](https://developer.mozilla.org/en-us/docs/Web/JavaScript/Reference/Functions/Arrow_functions), [сирці Arrow function expressions@GitHub](https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/functions/arrow_functions/index.md)

Нові зміни:
- [mdn/content@d4b12e2](https://github.com/mdn/content/commit/d4b12e290fce9ae43a9ae23b9b9c8a5812b82ebd)
- [mdn/content@b66ae46](https://github.com/mdn/content/commit/b66ae4647298ea7af4a2b58d67780df0795121e4)
- [mdn/content@61a8cdd](https://github.com/mdn/content/commit/61a8cddbde9257e05c55063985f28ac8d364082a)

### CODE REVIEW

**Comment by AdriandeCita** [comment]
є свіже оновлення https://github.com/mdn/content/commit/d4b12e290fce9ae43a9ae23b9b9c8a5812b82ebd

**Comment by undead404** [comment]
> є свіже оновлення [mdn/content@d4b12e2](https://github.com/mdn/content/commit/d4b12e290fce9ae43a9ae23b9b9c8a5812b82ebd)

Застосовано.

File: files/uk/web/javascript/reference/functions/arrow_functions/index.md [hunk 1]
```diff
@@ -206,6 +206,40 @@ Object.defineProperty(obj, 'b', {
 });
 ```
 
+Because a [class](/uk/docs/Web/JavaScript/Reference/Classes)'s body has a `this` context, arrow functions as [class fields](/uk/docs/Web/JavaScript/Reference/Classes/Public_class_fields) close over the class's `this` context and the `this` inside the arrow function's body will correctly point to the instance (or the class itself, for [static fields](/uk/docs/Web/JavaScript/Reference/Classes/static)). However, because it is a [closure](/uk/docs/Web/JavaScript/Closures), not the function's own binding, the value of `this` will not change based on the execution context.
```
Comment (AdriandeCita) [inline files/uk/web/javascript/reference/functions/arrow_functions/index.md:209]:
```suggestion
```

File: files/uk/web/javascript/reference/functions/arrow_functions/index.md [hunk 2]
```diff
@@ -206,6 +206,40 @@ Object.defineProperty(obj, 'b', {
...
+
```
Comment (AdriandeCita) [inline files/uk/web/javascript/reference/functions/arrow_functions/index.md:210]:
```suggestion
```

**Review by AdriandeCita — CHANGES_REQUESTED** [review]
трохи зайвого тексту

**Pull request merged**